spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -919,14 +919,6 @@ class CometSparkSessionExtensions
     }
 
     override def apply(plan: SparkPlan): SparkPlan = {
-
-      // Comet required off-heap memory to be enabled
-      if (!isOffHeapEnabled(conf) && !isTesting) {
```

Comment:
Looks like `isOffHeapEnabled` and `isTesting` are not used after PR. Should we delete those methods?

Reply:
Thanks. I have removed them now.